src/docs/markdown/caddyfile/directives/root.md

```diff
@@ -23,22 +23,29 @@ Note that a matcher token is usually required since the first argument is a path
 
 Set the site root to `/home/user/public_html` for all requests:
 
+(Note that a [wildcard matcher](/docs/caddyfile/matchers#wildcard-matchers) is required here because the first argument is ambiguous with a [path matcher](/docs/caddyfile/matchers#path-matchers).)
+
 ```
 root * /home/user/public_html
 ```
 
-(A [wildcard matcher](/docs/caddyfile/matchers#wildcard-matchers) is required in this case because the first argument is ambiguous with a [path matcher](/docs/caddyfile/matchers#path-matchers).)
-
 Set the site root to `public_html` (relative to current working directory) for all requests:
 
+(No matcher token is required here because our site root is a relative path, so it does not start with a forward slash and thus is not ambiguous.)
+
```

Comment:
FYI I moved these up because I feel it reads better if each example ends with their code block, otherwise it feels like the note has to do with the following example.